src/components/VirtualThought.tsx

```diff
@@ -107,7 +107,7 @@ const VirtualThought = ({
   const isVisible = zoomCursor || autofocus === 'show' || autofocus === 'dim'
   const shimHiddenThought = useDelayedAutofocus(autofocus, {
     delay: 750,
-    selector: autofocusNew => autofocus === 'hide' && autofocusNew === 'hide' && !!height,
+    selector: autofocusNew => ((autofocus === 'hide' && autofocusNew === 'hide') || (autofocus === 'hide-parent' && autofocusNew === 'hide-parent')) && !!height,
```

Comment:
I'll point out the comment a few lines above:

`// Only shim 'hide', not 'hide-parent', thoughts, otherwise hidden parents snap in instead of fading in when moving up the tree.`

Reply:
Hi @raineorshine Thanks for your feedback.
To smoothly animate the thoughts, change the duration time of the opacity transition of each thought from `150` to `1100` and adjust the transition-timing-function from `ease-in` to `ease-out`.

https://github.com/cybersemics/em/blob/f055c3a4a425ebe311e002c5950af5c1fa7906d4/src/components/Subthought.tsx#L132

https://github.com/cybersemics/em/blob/f055c3a4a425ebe311e002c5950af5c1fa7906d4/src/durations.config.ts#L28

Reply:
This animation intentionally uses a slow ease-out when the autofocus changes, and a fast ease-in otherwise, e.g. when thoughts are expanded. Altering the animation duration and timing function in the way you are suggesting will break the expected animation behavior.

## Suggested

https://github.com/user-attachments/assets/a6cf5e09-1929-48cc-8410-31144808832a

## Expected

`main`

https://github.com/user-attachments/assets/8c0f431b-1dab-4a9d-aad2-0097ef39c557